workflows/sarek.nf

```diff
@@ -948,10 +954,11 @@ workflow SAREK {
             // Gather used softwares versions
             ch_versions = ch_versions.mix(ANNOTATE.out.versions)
             ch_reports  = ch_reports.mix(ANNOTATE.out.reports)
-
         }
     }
 
+    ch_reports.collect().dump(tag:'ch_reports2')
+
```

Comment:
Is that still needed?

Reply:
No, as i mentioned above I still need to remove these dump statements and uncommented lines. It's more about the prefixes if you could have a look at that.

Reply:
:laughing:
I actually just read your initial comment now...
I don't see any issue wit the prefixes, but we might need to want to update docs at the same time.
So let's keep that in mind for https://github.com/nf-core/sarek/pull/610 cc @FriederikeHanssen